src/Serilog.Sinks.AzureDataExplorer/Serilog.Sinks.AzureDataExplorer.csproj

```diff
@@ -32,14 +31,22 @@ Improved options default values
     <AssemblyOriginatorKeyFile>Serilog.Sinks.AzureDataExplorer.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Serilog.Sinks.File" />
+  </ItemGroup>
   <ItemGroup>
     <None Include="..\..\README.md" Pack="true" PackagePath="$(PackageReadmeFile)" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Kusto.Ingest" Version="10.0.3" />
-    <PackageReference Include="serilog" Version="2.11.0" />
-    <PackageReference Include="Serilog.Sinks.PeriodicBatching" Version="2.3.1" />
+    <PackageReference Include="Microsoft.Azure.Kusto.Ingest" Version="11.2.2" />
+    <PackageReference Include="serilog" Version="2.12.0" />
+    <PackageReference Include="Serilog.Formatting.Compact" Version="1.1.1-dev-00944" />
```

Comment:
2 things

a) Look at using centralized package management
b) See if there are newer stable versions released
c) .NET standard extensions (may be even as a seperate branch)